src/main/resources/scripts/xslt/transformToPbCore.xsl

```diff
@@ -11,7 +11,7 @@
   <xsl:output encoding="UTF-8" method="xml" indent="yes" />
 
   <xsl:template name="pbcore_description">
-    <pbcore:pbcoreDescriptionDocument xsi:schemaLocation="http://www.pbcore.org/PBCore/PBCoreNamespace.html http://v2.pbcore.org/xsd/pbcore-2.0.xsd">
+    <pbcore:pbcoreDescriptionDocument xsi:schemaLocation="http://www.pbcore.org/PBCore/PBCoreNamespace.html  https://pbcore.org/xsd/pbcore-2.0.xsd">
```

Comment:
Ifølge id.kb.dk bruger KB PBCore version 2.1draft3 XML schema, men jeg ved ikke om det er korrekt eller om det har betydning her.

Reply:
Det Claus siger :-)

Reply:
Vi burde følge, hvad der står på id.kb.dk, men det virker underligt at bruge en draft udgave af skemaet... Især fordi den officielle version 2.1 er udkommet.
Har lige prøvet, at den virker faktisk med PBCore 2.1 skemaet ud af boksen, så forskellen på version 2.0 og 2.1 er nok ikke så stor, men det skal lige sikres med de data ansvarlige - kan være, at der skal ændres lidt i Cumulus eller øvrig dokumentation.
Tænker, at det må være en feature-request - eller en opgave, du kan tage med i bevaringsgruppen?

Reply:
Ja, synes også det var underligt med en draft version og det var derfor jeg ikke var sikker på det var korrekt. Som du siger lyder det som et feature-request. Jeg skal nok også tage det op i bevaringsgruppen.

Reply:
Så har jeg sendt forespørgsel til digitalbevarings gruppen. Det må være followup på review